cmd/framesd/framesd.go

```diff
@@ -45,24 +45,19 @@ var (
 
 func main() {
 	var config struct {
-		file        string
-		httpAddr    string
-		grpcAddr    string
-		showVersion bool
+		file     string
+		httpAddr string
+		grpcAddr string
 	}
 
 	flag.StringVar(&config.file, "config", "", "path to configuration file (YAML)")
 	flag.StringVar(&config.httpAddr, "httpAddr", ":8080", "address to listen on HTTP")
 	flag.StringVar(&config.grpcAddr, "grpcAddr", ":8081", "address to listen on gRPC")
-	flag.BoolVar(&config.showVersion, "version", false, "show version and exit")
 	flag.Parse()
 
 	log.SetFlags(0) // Show only messages
 
-	if config.showVersion {
-		fmt.Printf("%s version %s\n", path.Base(os.Args[0]), Version)
-		return
-	}
+	fmt.Printf("%s version %s\n", path.Base(os.Args[0]), Version)
```

Comment:
Just make sure it prints what you expect?

Reply:
tried building locally and copiy-tag it to a cluster and got "frames version latest" (which is the name when building locally)
i'm guessing that with an official release tag it will be right